packages/design-picker/src/utils/available-designs.ts

```diff
@@ -84,7 +90,16 @@ export function getAvailableDesigns(
 		),
 	};
 
+	if ( randomize ) {
```

Comment:
cc @tyxla — As agreed earlier today, this is how I replaced the `shuffle` function from `lodash` (see also changes to `client/landing/gutenboarding/index.tsx` in this PR)

Reply:
I'm happy with this approach, thanks for confirming 👍  

Just a minor nit: isn't this actually the Durstenfeld algorithm? I think it's considered a variation of Fisher-Yates, so it's fine to keep the inline comment, but maybe we could also mention that it's the Durstenfeld implementation.

Reply:
Thank you for noticing! I've updated the comment in https://github.com/Automattic/wp-calypso/pull/51633/commits/e80592d657931d53be5330bdc3d22fc7ebb80676